content/attacks/posts/2021-02-13-Alpha-Finance.md

```diff
@@ -0,0 +1,52 @@
+---
+date: 2021-02-13
+categories: DeFi, Protocol Exploit
+title: "Alpha Finance DeFi Hack: $37.5 Million Exploited"
+
+---
+
+## Summary
+
+On February 13, 2021, Alpha Finance, a decentralized finance (DeFi) project, was hacked, exploiting a vulnerability in the Alpha Homora v2 contract. The attacker successfully extracted $37.5 million from the project.
+
+## Attackers
+
+The identity of the attackers involved in the Alpha Finance DeFi hack remains unknown. Information using in this incident was not publicly available and was not accessible through the user interface, suggesting that the attacker may have possessed insider knowledge to carry out the attack.
+
+ ## [Losses](https://cryptobriefing.com/alpha-finance-suffers-37-5-million-loss-major-attack/)

```

Comment:
Don't put source links in section itself. It can be moved to the first sentence in the section